### ISSUE TITLE
Increase transparency about file changes between versions

### ISSUE THREAD

**Issue by elizabethdryad** [datadryad/dryad-product-roadmap#623]
When data files change after an item is published, the submitter should be required to enter a brief change history *to display publicly*, which will alert end users to differences between versions.  

This is different from the ‘describe edits made’ box (https://github.com/CDL-Dryad/dryad-product-roadmap/issues/540), because we don’t intend to expose every metadata change or author action required, but we do want to be transparent about how files change between versions. 

This change will also help curators know what needs to be checked when a dataset is resubmitted.

**Comment by ryscher** [comment]
This still seems like a good idea. @jleighherzog does it fit into your plans?

**Comment by jleighherzog** [comment]
@ryscher : Yeah. Actually, I posted a related ticket without realizing one existed (sorry). https://github.com/CDL-Dryad/dryad-product-roadmap/issues/2299#issue-1586282125

I'm not sure that I would make the changes between versions with text provided by the author _automatically_ public. We could, instead, include a step in our curation workflow to request that curators populate the Usage Notes with formatted, edited text that explains the difference between published versions (not sure what authors will include in the field and wouldn't want it to be automatically public without a check, might be safer (?) for us to add the text than fail to verify it). 

As Elizabeth mentions, useful only to require this for versioned submissions (previously published) AND only required for when the files uploaded are edited (Dryad and/or Zenodo).

@bryanmgee : Thoughts on this approach?

**Comment by bryanmgee** [comment]
@jleighherzog @ryscher I would also prefer not to post whatever authors put, which I suspect will largely be uninformative brevity like "I changed some files." Coming up with a standard practice, whether that's expanding the README or the Usage Notes would be better (bonus points if there is a way to tweak the system to recognize when files have been modified and to automatically flag authors to update these)

**Comment by jleighherzog** [comment]
Thanks, BG. Agreed.
